node/hubController.js

```diff
@@ -0,0 +1,222 @@
+/* jshint esversion: 6 */
+/* jshint node: true */
+'use strict';
+
+var q = require('q');
+var hubsConfig = require("./hubsConfig");
+
+class HubController {
+
+    constructor() {
+        this.supportedHubsCache = undefined;
+        this.OpenT2T = require('opent2t').OpenT2T;
+    }
+
+    /** 
+     * returns the list of supported hubs 
+     * format is as follows:
+     * [
+     *   {
+     *     id: "hubId",
+     *     name: hubName",
+     *     translator: "name of hub translator",
+     *     onboarding: "name of hub onboarding",
+     *     // this is the onboardingFlow object as defined by the hub's manifest.xml'
+     *     onboardingFlow: {}
+     *   }
+     * ]
+     */
+    supportedHubs(hubs, i) {
+        // use cache if we have it
+        if (this.supportedHubsCache != undefined) {
```

Comment:
>= undefine [](start = 37, length = 10)

quick question since i am getting confused with the syntax -- is it '!=' for ne or '!==' ?  and is 'undefined' a keyword or should it be in quotes?